src/routers/v2/DelaysRouter.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import ApplicationRouter from '../../appdev/ApplicationRouter';
+import RealtimeFeedUtils from '../../utils/RealtimeFeedUtils';
+
+class DelaysRouter extends ApplicationRouter<Object> {
+  constructor() {
+    super(['POST']);
+  }
+
+  getPath(): string {
+    return '/delays/';
+  }
+
+  async content(req): Promise<any> {
+    const delays = req.body.data;
+    const rtf = await RealtimeFeedUtils.fetchRTF();
+    const delaysPromises = delays.map(async (stopAndTripID) => {
+      const tripID = await stopAndTripID.tripID;
+      const stopID = await stopAndTripID.stopID;
```

Comment:
why do these need to be `await`ed?

Reply:
I actually added the awaits because both the linter and my terminal were giving me errors:
<img width="767" alt="Screen Shot 2019-10-06 at 1 55 24 PM" src="https://user-images.githubusercontent.com/13739525/66273334-f8b9fc00-e840-11e9-8959-3415621eff3f.png">
<img width="616" alt="Screen Shot 2019-10-06 at 1 55 18 PM" src="https://user-images.githubusercontent.com/13739525/66273335-fa83bf80-e840-11e9-8e32-5d2625db9885.png">

Reply:
i got rid of the variables instead and are just directly referencing the attributes now

Reply:
Here's a helpful article about destructuring https://wesbos.com/destructuring-objects/
It's something that is commonly used in JS!